package.json

```diff
@@ -1,16 +1,18 @@
 {
   "name": "assessment-backend",
   "version": "1.0.0",
-  "main": "index.js",
+  "main": "indexCorrected.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon indexCorrected.js"
   },
   "keywords": [],
-  "author": "",
+  "author": "Braden Toma",
   "license": "ISC",
   "dependencies": {
     "cors": "^2.8.5",
-    "express": "^4.18.1"
+    "express": "^4.18.1",
+    "nodemon": "latest"
```

Comment:
Adding nodemon is just a blessing to have. Checkout this [link](https://www.npmjs.com/package/nodemon) if you want to learn more about what it does. It's called in line 7 in this file before passing the argument of the starting file.